### PR TITLE
fix: bump adb to fix refreshGeoLocationCache command

### PR DESCRIPTION
**Pull request by KazuCocoa**
Apply https://github.com/appium/appium-adb/pull/623

I also appended description in mockLocationApp about how we can reset the mocked location info as a tip.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ appium:buildToolsVersion | The version of Android build tools to use. By default
 appium:skipLogcatCapture | Being set to `true` disables automatic logcat output collection during the test run. `false` by default
 appium:suppressKillServer | Being set to `true` prevents the driver from ever killing the ADB server explicitly. Could be useful if ADB is connected wirelessly. `false` by default
 appium:ignoreHiddenApiPolicyError | Being set to `true` ignores a failure while changing hidden API access policies. Could be useful on some devices, where access to these policies has been locked by its vendor. `false` by default.
-appium:mockLocationApp | Sets the package identifier of the app, which is used as a system mock location provider since Appium 1.18.0+. This capability has no effect on emulators. If the value is set to `null` or an empty string, then Appium will skip the mocked location provider setup procedure. Defaults to Appium Setting package identifier (`io.appium.settings`).
```
Comment (mykola-mokhnach) [inline README.md:105]: > Generally, termination of a mocked application resets the mocked location data.

I do not quite understand what the second sentence is about. Could you please explain?

Comment (mykola-mokhnach) [reply README.md:105]: I do not quite understand what the second sentence is about. Could you please explain?

Comment (KazuCocoa) [reply README.md:105]: Broadcast receiver features did not work after terminating io.appium.settings, so users should restart the process again to not break our features (via the broadcast receiver) when they stop the settings process by following this advic

Comment (mykola-mokhnach) [reply README.md:105]: Is there some example where process restart was needed? I am just trying to understand when the trick could/should be applied

Comment (KazuCocoa) [reply README.md:105]: Yea, I tried getting the device location after terminating the `io.appium.settings` via terminate_app. It returned an exception as below `/location` response. After activating the app again, the same command returned. I haven't dug into the io.appium.settings side, but potentially this is `LocationInfoReceiver`'s something issue that not returned any location info. (I initially thought something broadcast receiver stuff, but maybe location handling thing?)

Anyway, this example is what I tried to address to activate the terminated `io.appium.settings`.

```
[HTTP] --> GET /wd/hub/session/5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7/location
[HTTP] {}
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Calling AppiumDriver.getGeoLocation() with args: ["5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7"]
[debug] [ADB] Running '/Users/kazuaki/Library/Android/sdk/platform-tools/adb -P 5037 -s D0AA002182JC0202126 shell am broadcast -n io.appium.settings/.receivers.LocationInfoReceiver -a io.appium.settings.location'
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Encountered internal error running command: Error: Cannot parse the actual location values from the command output: Broadcasting: Intent { act=io.appium.settings.location flg=0x400000 cmp=io.appium.settings/.receivers.LocationInfoReceiver }
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Broadcast completed: result=0, data=""
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)]     at ADB.getGeoLocation (/Users/kazuaki/.appium/node_modules/appium-uiautomator2-driver/node_modules/appium-adb/lib/tools/settings-client-commands.js:287:11)
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)]     at AndroidUiautomator2Driver.getGeoLocation (/Users/kazuaki/.appium/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/network.js:168:43)
[HTTP] <-- GET /wd/hub/session/5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7/location 500 164 ms - 1017
[HTTP]
[HTTP] --> POST /wd/hub/session/5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7/appium/device/activate_app
[HTTP] {"bundleId":"io.appium.settings"}
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Calling AppiumDriver.activateApp() with args: ["io.appium.settings",null,null,null,"5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7"]
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Activating 'io.appium.settings'
[debug] [ADB] Running '/Users/kazuaki/Library/Android/sdk/platform-tools/adb -P 5037 -s D0AA002182JC0202126 shell cmd package resolve-activity --brief io.appium.settings'
[debug] [ADB] Running '/Users/kazuaki/Library/Android/sdk/platform-tools/adb -P 5037 -s D0AA002182JC0202126 shell am start-activity -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000 -n io.appium.settings/.Settings'
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=io.appium.settings/.Settings }
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Responding to client with driver.activateApp() result: null
[HTTP] <-- POST /wd/hub/session/5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7/appium/device/activate_app 200 217 ms - 14
[HTTP]
[HTTP] --> GET /wd/hub/session/5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7/location
[HTTP] {}
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Calling AppiumDriver.getGeoLocation() with args: ["5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7"]
[debug] [ADB] Running '/Users/kazuaki/Library/Android/sdk/platform-tools/adb -P 5037 -s D0AA002182JC0202126 shell am broadcast -n io.appium.settings/.receivers.LocationInfoReceiver -a io.appium.settings.location'
[debug] [ADB] Got geo coordinates: {"latitude":"37.3950608","longitude":"-122.0993036","altitude":"-5.5999999"}
[debug] [AndroidUiautomator2Driver@19c3 (5a1f7ae4)] Responding to client with driver.getGeoLocation() result: {"latitude":37.3950608,"longitude":-122.0993036,"altitude":-5.5999999}
[HTTP] <-- GET /wd/hub/session/5a1f7ae4-d4b2-4c97-9def-3bf113bb92e7/location 200 88 ms - 80
```

Comment (mykola-mokhnach) [reply README.md:105]: I assume https://github.com/appium/appium-adb/pull/624 should help to make this call more stable

Comment (KazuCocoa) [reply README.md:105]: I removed the sentence

File: README.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ appium:buildToolsVersion | The version of Android build tools to use. By default
...
+appium:mockLocationApp | Sets the package identifier of the app, which is used as a system mock location provider since Appium 1.18.0+. This capability has no effect on emulators. If the value is set to `null` or an empty string, then Appium will skip the mocked location provider setup procedure. Defaults to Appium Setting package identifier (`io.appium.settings`). Generally, termination of a mocked application resets the mocked location data.
```
Comment (mykola-mokhnach) [inline README.md:105]: Termination of a mock location provider application resets the mocked location data.

File: package.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.0.0",
-    "appium-adb": "^9.10.9",
+    "appium-adb": "^9.10.13",
```
Comment (mykola-mokhnach) [inline package.json:58]: could you please change to 9.10.14?

File: README.md (hunk 2 above)
Comment (KazuCocoa) [reply README.md]: ✅

File: package.json (hunk 3 above)
Comment (KazuCocoa) [reply package.json]: ✅

**Pull request merged**